test/powershell/engine/HelpSystem.OnlineHelp.Tests.ps1

```diff
@@ -0,0 +1,67 @@
+Describe 'Online help tests for PowerShell Core Cmdlets' -Tags "CI" {
+
+    $skipTests = [System.Management.Automation.Platform]::IsIoT -or [System.Management.Automation.Platform]::IsNanoServer -or [System.Management.Automation.Platform]::IsLinux
```

Comment:
why are you skipping IoT, NanoServer and Linux? when `BypassOnlineHelpRetrieval` is set, the code won't reach the `Process.Start` part anyway ...